### PR TITLE
Feature/effectiveness article

### PR DESCRIPTION
**Pull request by MrMichael3**

### CODE REVIEW

File: src/data/articles/systems/effectiveness.json [hunk 1]
```diff
@@ -1,4 +1,74 @@
...
   "title": "Effectiveness",
-  "paragraph": "Coming Soon!"
+  "paragraph": "Most of the units are effective against one or multiple units. The effective buff is applied on each fight for the attacker and the defender too. If each unit is effective against all enemies you can get a buff of +100%.",
+  "chapters": [
+    {
+      "subtitle": "Calculations",
+      "dynamicContents": [
+        {
+          "type": "text",
+          "content": "In a battle, the percentage of strength of each unit type is calculated (checkout {{ fight system # article/systems/fight }} for more details). To calculate the total effectiveness buff, add the effectiveness buff of each unit type. Take the percentage of total strength of a unit type and multiply it by the percentage of the enemy army it is effectively against."
```
Comment (ArminWindlin) [inline src/data/articles/systems/effectiveness.json]: 'effective' instead of 'effectively'

File: src/data/articles/systems/effectiveness.json [hunk 2]
```diff
@@ -1,4 +1,74 @@
...
+                "20%",
+                "100%",
+                "20%"
+              ]
+            ]
+          }
+        },
+        {
+          "type": "text",
+          "content": "\n<span style='color: violet'>Attacker:</span>\nBoth unit types of the attacker are effective against 80% of the defenders army. This leads to a total effectiveness buff of <strong>80%</strong> (32% + 48%) on this fight. The total attack power is raised from 25'000 to 45'000."
```
Comment (ArminWindlin) [inline src/data/articles/systems/effectiveness.json:65]: 'sends' instead of 'send'

File: src/data/articles/systems/effectiveness.json [hunk 3]
```diff
@@ -1,4 +1,74 @@
...
+        },
+        {
+          "type": "text",
+          "content": "<span style='color: darkgreen'>Defender:</span>\nThe Nyxi is effective against both enemy types which gives him a 100% effectiveness buff. But because the Nyxi makes just 20% of the defenders army and the Elder Spirit isn't effective, the total effectiveness is only <strong>20%</strong>. This raises the total defense power from 30'000 to 36'000. On this fight the attacker would win (other buffs such as field buff or runes where ignored for simplicity)."
```
Comment (ArminWindlin) [inline src/data/articles/systems/effectiveness.json]: 'were' instead of 'where' in the last sentence

**Review by ArminWindlin — CHANGES_REQUESTED** [review]
The image you added doesn't seem to be used anywhere.

Can you add this article to the wiki overview page?

I can recommend Grammarly to check for grammar. It can be installed for free as a chrome extension. Then you can just copy the text and paste it into some web editor (for example a google docs file) and it will show you most mistakes.

File: src/data/articles/systems/effectiveness.json [hunk 4]
```diff
@@ -1,4 +1,74 @@
...
+              "<span style='color: violet'>Druid Naki</span>",
+              "<span style='color: violet'>Guard Naki</span>",
+              "<span style='color: darkgreen'>Elder Spirit</span>",
+              "<span style='color: darkgreen'>Nyxi</span>"
+            ],
+            "columnHeaders": [
+              "Strength",
+              "percentage (%) of army",
+              "effective against %",
+              "total effectiveness"
```
Comment (ArminWindlin) [inline src/data/articles/systems/effectiveness.json]: I suggest also start them with a capital letter

**Comment by MrMichael3** [comment]
Thanks Armin
grammarly is very helpful, already using it. forgot it on this page...
didn't removed image from the assets, done now

**Review by ArminWindlin — APPROVED** [review]

**Pull request merged**